macOS/DuckDuckGo/TabBar/ViewModel/TabCollectionViewModel.swift

```diff
@@ -685,40 +689,6 @@ final class TabCollectionViewModel: NSObject {
         delegate?.tabCollectionViewModelDidMultipleChanges(self)
     }
 
-    func removeAllTabsAndAppendNew(forceChange: Bool = false) {
-        guard changesEnabled || forceChange else { return }
-
-        tabCollection.removeAll(andAppend: Tab(content: .newtab, burnerMode: burnerMode))
-        selectUnpinnedTab(at: 0, forceChange: forceChange)
-
-        delegate?.tabCollectionViewModelDidMultipleChanges(self)
-    }
-
-    func removeTabsAndAppendNew(at indexSet: IndexSet, forceChange: Bool = false) {
-        guard !indexSet.isEmpty, changesEnabled || forceChange else { return }
-        guard let selectionIndex = selectionIndex?.item else {
-            Logger.tabLazyLoading.error("TabCollection: No tab selected")
-            return
-        }
-
-        tabCollection.removeTabs(at: indexSet)
-        if tabCollection.tabs.isEmpty {
-            tabCollection.append(tab: Tab(content: .newtab, burnerMode: burnerMode))
-            selectUnpinnedTab(at: 0, forceChange: forceChange)
-        } else {
-            let selectionDiff = indexSet.reduce(0) { result, index in
-                if index < selectionIndex {
-                    return result + 1
-                } else {
-                    return result
-                }
-            }
-
-            selectUnpinnedTab(at: max(min(selectionIndex - selectionDiff, tabCollection.tabs.count - 1), 0), forceChange: forceChange)
-        }
-        delegate?.tabCollectionViewModelDidMultipleChanges(self)
-    }
```

Comment:
I've noticed these two methods are 💯 unused. Would it be okay if we dropped them?